backup_dont_use/js_qml_insides/DivideMeasures.js

```diff
@@ -95,6 +98,8 @@ for(var measure_id = 0; measure_id < measures.length; measure_id++){
         }
     }
     add_time_to_fun(funList, upperTime)
+    console.log(funList)
```

Comment:
Remove or use Utils.log

Reply:
backup_dont_use :) Did not realize that I pushed this